README.md

```diff
@@ -235,58 +235,69 @@ For headaches, take Ibuprofen
 The gobin command installs/runs main packages.
 
 Usage:
-	gobin [-m] [-r|-p] [-n|-g] packages [run arguments...]
+	gobin [-m] [-run|-p] [-nonet|-u] packages [run arguments...]
 
-The packages argument to gobin is similar to that of the go tool (in module
-mode) with the additional constraint that the list of packages must be main
-packages.
+The packages argument to gobin is similar to that of the go get command (in
+module mode) with the additional constraint that the list of packages must be
+main packages. Each argument takes the form $main_pkg[@$version].
 
 By default, gobin is said to operate in global mode. If the -m flag is provided
```

Comment:
I'm still not sure whether using "global mode" in this way is a good idea, because it sounds like it might relate to where the binaries end up rather than how dependencies are resolved (I had to specifically go back to check that it didn't do that).

How about leaving out any mention of "global mode" and saying something like:

    By default, gobin will use the module of the main package itself to resolve its
    dependencies, unless the `-m` flag is specified, in which case dependencies
    will be resolved using the current module (as given by `go env GOMOD`).

    This means that as `gobin $package@v1.2.3` is a repeatable way to install
    an exact version of a binary (assuming it has an associated `go.mod` file).

Reply:
Also, to add a "note to self" - either we need a `-d` flag or we make mention of the fact that `-p` prevents the install. The former feels more natural

Reply:
I like your suggestion re dropping "global".